### ISSUE TITLE
[UWP][Accessibility] 'Close' control is not accessible through keyboard at left panel.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3838]
# Platform

* UWP

# Version of SDK

master

# Details

Repro Steps:
1. Launch the application.
2. New screen starts appearing. 
3. Navigate to various json file like Activity Update.JSON, Agenda.JSON etc. at left panel.
4. Navigate to 'Close' control for each json file.
5. Verify 'Close' control by using keyboard.

Actual Result: 
'Close' control is not accessible through keyboard appearing over each json file like Agenda.JSON, Agenda.JSON at left panel which will impact the functionality of relevant control. Not only this, user won't be able to perform functionality over such control.

Expected Result: 
'Close' control should be accessible through keyboard appearing over each json file like Agenda.JSON, Agenda.JSON at left panel so that end user could easily perform the relevant functionality.
